.travis.yml

```diff
@@ -17,6 +17,8 @@ stages:
   - check nightlies
 
 jobs:
+  allow_failures:
+    - os: osx
```

Comment:
Feels bad, but I guess it's easier..

Reply:
Ah, it was for my debugging, dropped. But actually, if a test fails on the way, stage builds don't run further stages. It's not useful a bit.